python-project-template/.github/workflows/{% if preferred_linter != 'none' %}linting.yml{% endif %}.jinja

```diff
@@ -37,4 +37,4 @@ jobs:
       uses: psf/black@stable
       with:
         src: ./src
-{% endif %}
+{% endif -%}
```

Comment:
Same a prior comment: Wouldn't this remove the new line at the end of the file when the template is hydrated?

Reply:
it removes the following newline (after the %} in the same line) which leaves the newline at the end of ./src
which is what we want